### ISSUE TITLE
Filter out logging of login attacks

### ISSUE THREAD

**Issue by markbyrn** [TotalFreedom/TotalFreedomMod#543]
We now have a persistent attack of rcon logins and it's filling up the logs and telnet stream.  For example:

:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47

In the past, we had filtering for failed login attempts although not specifically rcon logins. Failing to fix this means we can't really use Telnet very effectively and the logs get very large.

**Comment by markbyrn** [comment]
If there's a non-coding solution, let me know.

**Comment by JeromSar** [comment]
@markbyrn This might be TwitterBot checking if the server is online.

**Comment by Wild1145** [comment]
You could filter it from telnet and just filter anything Rcon, but you couldnt filter it from the logs (That I can think of) as thats vanilla functionality.

**Comment by markbyrn** [comment]
no, it's not a twitter bot thing because the same IP is doing this to:

:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Client connected: 69.36.188.47
:[23:31:30 INFO]: [BukkitTelnet] Client connected: 69.36.188.47
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Client connected: 69.36.188.47
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)
:[23:31:30 INFO]: [BukkitTelnet] Client connected: 69.36.188.47
:[23:31:30 INFO]: [BukkitTelnet] Closing connection: 69.36.188.47 (markbyronSuxButt)

\

**Comment by markbyrn** [comment]
Multiplay said they would block that IP on their end but obviously the attacker can change IPs.

**Comment by JeromSar** [comment]
@markbyrn I'd suggest changing the port for now. Likewise, an admin is probably behind this attack, considering he/she knows the BukkitTelnet port. Additionally, I'd say the attacker is tech-savvy considering the complexity of the attack.

**Comment by Wild1145** [comment]
@JeromSar The problem is, if it is indeed a current admin, changing the port wont do pot, because they will just change the port they are attacking on.

**Comment by StevenLawson** [comment]
I suggest adding functionality to throttle connections from the same IP and
silently / automatically block this kind of thing.

**Comment by StevenLawson** [comment]
Not to mention its pretty easy to scan for an open telnet port.

**Comment by markbyrn** [comment]
This is not a Telnet problem - it's RCON login attempt and the only issue with Telnet is that it shows the login attempt and spams.  Can't change the RCON port as far as I know.

**Comment by JeromSar** [comment]
@markbyrn You can change the RCon port too! It's the `admin port` in ClanForge. As far as I can see, both Rcon and Telnet are affected:

```
:[23:31:30 INFO]: [BukkitTelnet] Client connected: 69.36.188.47
```
```
:[00:30:53 INFO]: Rcon connection from: /69.36.188.47
```

**Comment by Wild1145** [comment]
@JeromSar I believe Clanforge relies on RCON though, so if you change it, the Clanforge console might break...

**Comment by JeromSar** [comment]
@Wild1145 Hmm, fair point..

**Comment by Wild1145** [comment]
@JeromSar Its just something to double check. Its one of the drawbacks of having to use Clanforge :/ It might get the port from the config then attempt to connect, or it might only allow default port, Multiplay might be able to help @markbyrn clear that up if you do think changing port would be a good idea (and it probably would be, especially if its on default port, and as admins dont have access to see the RCON port (Seniors might), then its a good idea)

**Comment by ChargedCreeper** [comment]
@Wild1145 All you need to see the admin port is clanforge access. So Telnet Clan can see it too. Also there are some Minecraft modded clients that have port scanners.

**Comment by Wild1145** [comment]
@ChargedCreeper Okay, as I say I dont know how Clanforge works other than the console uses RCON, so it is just something I recommend checking still works before you do anything to major ;)

**Comment by xDestroyer217** [comment]
And this also seems that this isn't the only server being attacked with that spam..
